Ricardo rdz g cleaner install

RicardoRdzG opened the pull request:
Fixed several bugs in install script and added code for cleaner install

RicardoRdzG committed:
Update install.sh

RicardoRdzG committed:
Test first for initialization code

Instead of blindly adding initialization code to all .bash files look for it first and do nothing if already present

RicardoRdzG committed:
Keep one time use functions in install.sh

Avoid scattering code in different files to make it easy to understand and avoid extra code

RicardoRdzG committed:
Load aliases as part of initialization code

purplesyringa commented on install.sh:
```diff
@@ -1,5 +1,37 @@
+function ws-success {
+  echo Win-sudo successfully installed!
+  exec bash
+  alias fresh="exec bash"
```
Shouldn't these 2 lines be swapped?

RicardoRdzG replied:
probably, I just moved those over from the other file I removed

RicardoRdzG committed:
updated with the latest install aliases

updated with the latest install aliases
test first if install dir exist before removing it

RicardoRdzG committed:
Merge branch 'master' into RicardoRdzG-cleaner-install

RicardoRdzG committed:
properly source aliases

imachug commented:
Thank you!
